Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/PixRedistMiddleware.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include "pch.h"

```

Comment:
Is this sample code like GetLatestWinPixGpuCapturerPath_Cppp17 from [here](https://devblogs.microsoft.com/pix/taking-a-capture/)?

Reply:
Yeah this is something I should add comments to clarify in the code. 

First off, this sample code uses the redist build instead of the inbox one, it will try to dynamically load the redist WinPixEventRuntime.dll and map required methods, the example code from PIX landing page looks like it assumes the inbox version is used, therefore loading WinPixEventRuntime.dll was not needed.

Secondly, the example shown on PIX landing page is for manual capturing. User would still need to manually attach their process to PIX app to do a capture. This sample code shows how to do capture programmatically 100% (given then PIX is still installed)